input/pagecontent/changes.md

```diff
@@ -76,11 +76,9 @@ This change log documents the significant updates and resolutions implemented fr
 - Updated mapping of AUCDI Procedure Completed Event to be represented by Procedure only [au-fhir-core #147](https://github.com/hl7au/au-fhir-core/issues/147).
 - Updated mapping of AUCDI Medication Use Statement to be represented by MedicationStatement only [au-fhir-core #145](https://github.com/hl7au/au-fhir-core/issues/145).
 - Made the following changes to AU Core Condition:
-  - Removed Must Support from the following elements in AU Core Condition:
-    - Condition.asserter [FHIR-45019](https://jira.hl7.org/browse/FHIR-45019), [FHIR-45127](https://jira.hl7.org/browse/FHIR-45127), [FHIR-45083](https://jira.hl7.org/browse/FHIR-45083)
-    - Condition.recorder [FHIR-45019](https://jira.hl7.org/browse/FHIR-45019), [FHIR-45082](https://jira.hl7.org/browse/FHIR-45082)
-    - Condition.encounter [FHIR-45189](https://jira.hl7.org/browse/FHIR-45189)
-  - Condition [FHIR-45058](https://jira.hl7.org/browse/FHIR-45058) additional narrative with guidance on *no known problem* coding. 
```

Comment:
restore change log line for no known problem coding but make better so it is "Added profile-specific implementation guidance on representing no known problem [FHIR-45058](https://jira.hl7.org/browse/FHIR-45058)"

Reply:
Fixed.

Reply:
Fix the copy paste spelling error. "problems" instead of "problem: